WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController+OnboardingPrompt.swift

```diff
@@ -16,32 +16,26 @@ extension MySiteViewController {
         switch option {
         case .stats:
             // Show the stats view for the current blog
-            // We have to switch to the "Site Menu" first
-            switchTab(to: .siteMenu)
-
-            // Show the stats tab
-            blogDetailsViewController?.showStats(from: .button)
+            if let blog = blog {
+                StatsViewController.show(for: blog, from: self)
+            }
         case .writing:
             // Open the editor
             let controller = tabBarController as? WPTabBarController
             controller?.showPostTab(completion: {
                 self.startAlertTimer()
             })
-        case .notifications:
-            // Open the notifications tab
-            let controller = tabBarController as? WPTabBarController
-            controller?.showNotificationsTab()
-        case .reader:
-            // Open the reader tab
-            let controller = tabBarController as? WPTabBarController
-            controller?.showReaderTab()
+
         case .showMeAround:
             // Start the quick start
             if let blog = blog {
-                QuickStartTourGuide.shared.setup(for: blog)
+                QuickStartTourGuide.shared.setup(for: blog, type: .existingSite)
```

Comment:
We should pass this type only if the Feature Flag `quickStartForExistingUsers` is enabled, else pass `.newSite`

Reply:
Added the feature flag check for the quick start here: https://github.com/wordpress-mobile/WordPress-iOS/commit/6a613892b24d0208707d1b491fea5c894014451b